pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -1,27 +1,50 @@
-{ lib, stdenv, fetchurl, sconsPackages, boost, gperftools, pcre-cpp, snappy, zlib, libyamlcpp
-, sasl, openssl, libpcap, curl, Security, CoreFoundation, cctools }:
+{ lib
+, stdenv
+, fetchurl
+, sconsPackages
+, boost
+, gperftools
+, pcre-cpp
+, snappy
+, zlib
+, libyamlcpp
+, sasl
+, openssl
+, libpcap
+, python27
```

Comment:
Can we get rid of python2?

Reply:
Haven't touched that part, but it does seem that it is redundant as it is not used anywhere in the rest of the code. I'll remove it.